selectrum.el

```diff
@@ -1366,15 +1366,14 @@ shadows correctly."
                  (let ((candidate-paths
                         (mapcar (lambda (path)
                                   (propertize
-                                   (file-name-base path)
+                                   (file-name-base
+                                    (file-name-sans-extension path))
```

Comment:
Out of curiosity, do you know what causes the `.el` candidates? I don't see them but I'm willing to believe they show up under some configurations.

Reply:
No idea where they are coming from but I see them, too.